overlord/snapstate/handlers.go

```diff
@@ -202,7 +217,7 @@ func (m *SnapManager) installOneBaseOrRequired(st *state.State, snapName string,
 	}
 
 	// not installed, nor queued for install -> install it
-	ts, err := Install(st, snapName, defaultBaseSnapsChannel, snap.R(0), userID, Flags{})
+	ts, err := Install(st, snapName, channel, snap.R(0), userID, Flags{})
```

Comment:
Should we log the fact that SNAPD_BASES_CHANNEL env var is set and != stable somewhere so that we know that system is "tainted" that way?

Reply:
we don't do that with any other vars,  also it should be visible with SNAPD_DEBUG_HTTP turned on